mne/report.py

```diff
@@ -977,6 +978,13 @@ def __init__(self, info_fname=None, subjects_dir=None,
             raise TypeError('raw_psd must be bool or dict, got %s'
                             % (type(raw_psd),))
         self.raw_psd = raw_psd
+
+        # Dictionary containing elements included in head
+        self.include = OrderedDict()
+        # Counter for labeling custom css/js additions to self.include
+        self.custom_css_count = 0
+        self.custom_js_count = 0
```

Comment:
this is a public attribute that is not documented. Ideally you want to make anything that is not user facing to be private